src/Microsoft.AspNet.Identity/SignInManager.cs

```diff
@@ -406,8 +405,12 @@ public virtual AuthenticationProperties ConfigureExternalAuthenticationPropertie
         /// <returns></returns>
         protected async virtual Task<bool> LogResultAsync(bool result, TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
         {
-            Logger.WriteInformation(Resources.FormatLoggingSigninResult(Resources.FormatLoggingResultMessage(methodName,
-                await UserManager.GetUserIdAsync(user)), result));
+            // Check if log level is enabled before creating the message.
```

Comment:
This is even more clear in the derived result case, when they extend SignInResult with an additional failure, they'd have to override both SignInResult.ToString and SignInManager.LogResult today, instead of just overriding SignInResult.ToString to add logic for any new status codes, and then deferring to the parent behavior